sys/dev/netmap/netmap_mem2.c

```diff
@@ -1149,6 +1149,29 @@ netmap_mem_bufsize(struct netmap_mem_d *nmd)
     (netmap_obj_offset(&(n)->pools[NETMAP_BUF_POOL], (v)) / NETMAP_BDG_BUF_SIZE(n))
 #endif
 
+#ifdef ATL_CHANGE
+uint32_t
+netmap_ext_buf_malloc(struct netmap_adapter *na)
+{
+	struct netmap_mem_d *nmd = na->nm_mem;
+	uint32_t pos = 0;
```

Comment:
should we potentially pass both these as inputs to function so that they can both be returned, or is the 'pos' not that important?

Reply:
pos is a performance helper ... because allocating buffers requires scanning for the first free buffer, when allocating lots of buffers you can use pos to let the second allocation start from where the first one ended ... given we are using this only at init time (not during packet processing) i think this ok. The other approach would be do create a fn to malloc multiple buffers and return them in a list ... maybe later?

Reply:
Yep sounds good.